extensions/amp-story-interactive/0.1/amp-story-interactive-slider.css

```diff
@@ -95,6 +99,80 @@
   border-bottom-right-radius: 50% !important;
 }
 
+.i-amphtml-story-interactive-slider-average-bubble {
+  position: absolute !important;
+  transform: translate(-50%, -50%) scale(0) !important;
+  width: 2.8em !important;
+  height: 2.8em !important;
+  top: 50% !important;
+  left: var(--average) !important;
+  text-align: center !important;
+  line-height: 1em !important;
+  font-weight: 300 !important;
+  color: black !important;
+  z-index: 1 !important;
+}
+
+.i-amphtml-story-interactive-slider-average-bubble::after {
```

Comment:
We'll need to style the indicator appropriately, for which we could use `:before` and `:after` elements, but I don't think this pseudo-element should be used for this purpose (seems like you're adding a pointy end to a circle?)